### PR TITLE
Added an improvement in the search of the reference file

### LINKED ISSUE #3: kssh search on all my account, not in the directory path context
kssh current uses :
`find /home/myAccount/ -name srv_tmpl.yml` 

This is too heavy for large home dir.

On the other side, the idea was more on where you are in the current directory, try to find out if we find that srv_tmpl.yml and not all.

Ex:
```bash
$ cd /home/myaccount/src/myproject/subdir1/subdir2 
$ pwd
/home/myaccount/src/myproject/subdir1/subdir2
$ kssh --details
```
in this example, kssh will search for srv_tmpl.yml in the following order:
1. /home/myaccount/src/myproject/subdir1/subdir2/srv_tmpl.yml ?
2. /home/myaccount/src/myproject/subdir1/srv_tmpl.yml ?
3. /home/myaccount/src/myproject/srv_tmpl.yml ?
4. /home/myaccount/src/srv_tmpl.yml ?
5. /home/myaccount/srv_tmpl.yml ?
6. /home/srv_tmpl.yml ?
7. /srv_tmpl.yml ?

first found, first read.

We can imagine, if we are in git repo:
```bash
$ cd /home/myaccount/src/myproject/subdir1/subdir2 
$ pwd
/home/myaccount/src/myproject/subdir1/subdir2
$ git rev-parse  --show-toplevel
/home/myaccount/src/myproject
$ kssh --details
```
in this example, kssh will search for srv_tmpl.yml in the following order:
1. /home/myaccount/src/myproject/subdir1/subdir2/srv_tmpl.yml ?
2. /home/myaccount/src/myproject/subdir1/srv_tmpl.yml ?
3. /home/myaccount/src/myproject/srv_tmpl.yml ?

first found, first read.

### PR DESCRIPTION
**Pull request by cppmx**
With this change the search of the reference file no longer uses the **find** command.

Please test it.

fix #3

### CODE REVIEW

File: sh/kssh [hunk 1]
```diff
@@ -39,13 +39,24 @@ fi
...
+for (( ; ; ))
 do
-  TEMPVAR=$(dirname $path_to_test)
-  if stringContains $TEMPVAR $('pwd')
-  then
-    REPOSITORY_PROFILES=$TEMPVAR/services_information/servers/
+  # Let's check if the reference file is in the current directory
+  if [[ -f $CURRENT_DIR/srv_tmpl.yml ]]; then
+    # If so, then assign that directory in the propper variable
+    REPOSITORY_PROFILES=$CURRENT_DIR/services_information/servers/
```
Comment (clarsonneur) [inline sh/kssh]: `.../services_information/servers/` is normally defined in the srv_tmpl.yml file directly. we cannot assume to be in that path everytime.

**Review by clarsonneur — CHANGES_REQUESTED** [review]
Looks good. But need one more fix, to be ok.

File: sh/kssh (hunk 1 above)
Comment (cppmx) [reply sh/kssh]: You're right. You know, that's something I wanted to do from the first version, but I forgot to remove the hard code.

**Comment by cppmx** [comment]
I have already uploaded the change.

**Review by clarsonneur — APPROVED** [review]
LGTM
Thank you!

**Pull request merged**